### DISCUSSION TITLE
Found a bug in the `jsonwebtoken` library thanks to `fast-check`!

### DISCUSSION THREAD

**Discussion by ej-shafran** [dubzzz/fast-check#4555] [Show and tell]
The property tests I added to a library I'm maintaining would crash every once in a while in an unexplained manner. I wasn't sure whether the error was my fault or not, but writing a simple property-based test with `fast-check` for `jsonwebtoken` revealed the problem!

You can see more [in this issue](https://github.com/auth0/node-jsonwebtoken/issues/945).

**Comment by dubzzz** [comment]
Whaou, so great 🤩

Please feel free to advertise it on this page of the official documentation: https://fast-check.dev/docs/introduction/track-record/. Such bugs on such commonly used blocks of the ecosystem are highly valuable and clearly underline the value of property based testing outside of testing alone.

**Reply by ej-shafran** [reply]
Opened a [pull request](https://github.com/dubzzz/fast-check/pull/4677) :)
